config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
+  root to: 'texts#index'
+  resources :texts
```

Comment:
このように不要なルーティングを作成しないのが理想です。こちらの修正をプッシュされたらマージしてしまって下さい！

```suggestion
  resources :texts, only: [:index, :show]
```